src/sentry/static/sentry/app/views/groupMerged/mergedItem.jsx

```diff
@@ -74,53 +73,51 @@ const MergedItem = createReactClass({
     GroupingActions.toggleUnmerge([fingerprint, event.id]);
   },
 
+  handleCheckClick() {
+    // noop because of react warning about being a controlled input without `onChange`
```

Comment:
Should noop just be the default onChange value so we don't have to bother with this in future?

Reply:
I think the warnings are helpful and this is more of an edge case. I think we should re-evaluate if we see more of these cases though.